app/channels/application_cable/conversation_channel.rb

```diff
@@ -0,0 +1,23 @@
+class ConversationChannel < ApplicationCable::Channel
+  def subscribed
+    stop_all_streams
+    Conversation.where(user1: current_user0).or(Conversation.where(user2: current_user)).find_each do |conversation|
+      stream_from "conversations_#{conversation.id}"
```

Comment:
```suggestion
      stream_for "conversations_#{conversation.id}"
```

I think [stream_from](external_repositories) is for broadcasting.

Reply:
Fixed